examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1640 @@
+{
```

Comment:
<p>What does <code>we neglect it</code> mean? what did we neglect? the items that are in validation set but not in train set? if yes, please write down clearly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>